env_vars/site_sthlm_env_staging.yml

```diff
@@ -0,0 +1,18 @@
+
+# statusdb
+statusdb_creds: statusdb_creds_stage.yml
+statusdb_url: ngi-statusdb-dev.scilifelab.se
+statusdb_port: 5984
+
+orderportal_creds: orderportal_credentials.yaml
```

Comment:
This seems to be the same for staging and production, so I would put it in site_sthlm_env_all.yml, but maybe it doesn't matter that much.